pyiron_contrib/protocol/compound/nudged_elastic_band.py

```diff
@@ -182,6 +180,15 @@ def _get_energies(self, frame=None):
             return self.graph.calc_static.archive.output.energy_pot.data[frame]
 
     def plot_elastic_band(self, ax=None, frame=None, plot_kwargs=None):
+        """
+        Plot the string at an input frame. Here, frame is a dump of a step in the run. The number of dumps
+            can be specified by the user while submitting the job, as:
+
+        <job_name>.set_output_whitelist(**{'calc_static': {'energy_pot': <dump every these many steps>}}).
```

Comment:
There is a way to make the examples be formatted as code in docstrings. Something like ">>> example_code", but check some of the recent pyiron PRs to see (Marvin and I think Sam have both been taking strong advantage of this feature).

Reply:
I will check it out.

Reply:
I could not find the code after 20 mins of searching for it. @pmrv could you share an example, maybe?

Reply:
In https://github.com/pyiron/pyiron_base/blob/862be30490fda1b94b2e83788aa7c5ed45bf7d4d/pyiron_base/generic/util.py#L83 Marvin used the ">>> example_code" syntax like Liam suggested.

Reply:
More examples [here](https://docs.python.org/3/library/doctest.html?highlight=doctest#module-doctest).  When possible, I like to make sure the example code also works as ad-hoc tests, but outside utility functions/classes that often too complicated so it's not a must.